
""" 
    The PokemonNotFoundError class is a subclass of the built-in Exception class in Python. It is a custom exception class that is used to handle the case when a Pokemon is not found.


pass: This is a placeholder statement. It does nothing. It is used when a class definition is required, but the class does not need to have any methods.
So, the PokemonNotFoundError class is a custom exception class that is used to handle the case when a Pokemon is not found. It is a subclass of the Exception class, which means it can be raised and caught like any other exception in Python.

The pass statement in the class definition is used to indicate that the class does not have any methods. It is wrapper for and concrete type of error
"""


class PokemonNotFoundError(Exception):
    pass
